src/parsers/parallel.cr

```diff
@@ -17,7 +17,7 @@ module Mint
           results = many { statement(Ast::Statement::Parent::Sequence) || comment }.compact
 
           raise ParallelExpectedStatement if results
-                                               .reject(&.is_a?(Ast::Comment))
+                                               .reject(Ast::Comment)
```

Comment:
`reject!`?

Reply:
I don't think we want to modify the receiver in this case.

Reply:
yes, you are right, sorry.